### DISCUSSION TITLE
Welcome to Circle_finder Discussions!

### DISCUSSION THREAD

**Discussion by pk7zuva** [pk7zuva/Circle_finder#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sarlago** [comment]
Hi,
I am trying to apply your code to identify eccDNA from ATACseq data sequenced with PE 100bp chemistry.
Following your instructions I run just the script circle_finder-pipeline-bwa-mem-samblaster.sh with the required arguments.
Actually I have some questions: 
1) Is it correct to just run this script and not the other you mentioned in the tutorial for either circle-DNA enriched or reads shorter than 75bp, correct? This script does not use bowtie2, so I don't need to have it installed, right?
2) Should I trim adapters from my reads before running Circle Finder?
3) The output file I should look at is *hg38.microDNA-JT.txt, which is a bed file with four columns in my case, what does the fourth column value indicate? And why don't I have a 6 columns output as you stated in the README tutorial?

Thank a lot for your help.
